ui-community/src/components/shared/apollo-client-links.tsx

```diff
@@ -0,0 +1,69 @@
+import { ApolloClient, ApolloLink, DefaultContext, InMemoryCache, from } from '@apollo/client';
+import { BatchHttpLink } from '@apollo/client/link/batch-http';
+import { AuthContextProps } from 'react-oidc-context';
+import { removeTypenameFromVariables } from '@apollo/client/link/remove-typename';
+import { setContext } from '@apollo/client/link/context';
+
+
+export const client = new ApolloClient({
```

Comment:
1 is implemented. 2 is not implemented because apollo-connection must specify the default and updated config for ApolloClient. This code file only comtains custom and reusable apollo links